addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1930,6 +1930,7 @@ const Wysiwyg = Widget.extend({
                 title: _t('Link'),
                 description: _t('Add a link.'),
                 fontawesome: 'fa-link',
+                allowInInlineRoot: true,
```

Comment:
What I had in mind was more like `condition: <predicate>` and we would have a predicate util which would be is `isInBlock`, so that the commands definitions would read:
```js
condition: isInBlock,
```

Reply:
I can try that indeed, it would be more extensible. 
I'm just not sure yet if this will be possible easely at command definition outside of the odooeditor scope.

will try

Reply:
Are you sure this can't be done with the already existing [`isDisabled` property](https://www.odoo.com/documentation/16.0/developer/reference/frontend/odoo_editor.html#reference) ?
By the way if you change the API, please make sure to update the doc, it would be too bad for it to be outdated.

Reply:
oh yes, didn't knew this was a thing. I'll use that.